composer.json

```diff
@@ -10,6 +10,7 @@
         }
     ],
     "require": {
+        "ezsystems/doctrine-dbal-schema": "^1.0@dev",
```

Comment:
Do we use `doctrine-dbal-schema` within this package?

Reply:
We don't, directly. This is advised by Composer doc:
```
If one of your dependencies has a dependency on an unstable package you need to explicitly require it as well, along with its sufficient stability flag.
```
`doctrine-dbal-schema:1.0` (a dependency of ezpublish-kernel) exists as `dev` and `beta` versions, meaning no stable versions.

Problem is, that by using:
```
    "minimum-stability": "dev",
    "prefer-stable": true,
```
Composer will install ezpublish-kernel:8.0.0-beta1, which is quite outdated now (for example. all the Events refactoring happened after that)

When using just `minimum-stability": "dev",`we would get all Symfony dev dependencies (too many of them).

Without specyfing `doctrine-dbal-schema` (and removing the prefer-stable, minimum stability options) executing `composer install` won't work for this package.

See: https://github.com/ezsystems/ezplatform-richtext/pull/57 , where the same issue happened (and also many other packages).

I've added prefer-stable, minimum stability options when work for 3.0 has started as it worked then, but when we've released the beta it stopped working as we'd like. 

My idea is to remove the unneeded dependencies before we release 3.0 final, I don't like this solution too much to be honest, so all suggestions are welcome.